variants/portduino/platformio.ini

```diff
@@ -6,6 +6,7 @@ build_flags = ${portduino_base.build_flags} -O0 -I variants/portduino
   -I /usr/include
   !pkg-config --libs libulfius --silence-errors || :
   !pkg-config --libs openssl --silence-errors || :
+  -li2c
```

Comment:
Doesn't this belong in `arch/portduino/portduino.ini`?

Reply:
Yes, it probably does. Good spot!